mapcomposer/app/static/externals/gxp/src/script/plugins/ChartReporting.js

```diff
@@ -48,7 +57,42 @@ gxp.plugins.ChartReporting = Ext.extend(gxp.plugins.Tool, {
             msg: "Not yet implemented"}
          );
     },
-    
+
+    init : function(target) {
+        this.target = target;
+        gxp.plugins.ChartReporting.superclass.init.call(this, target);   
+        if(!this.geoStoreClient) {
+            this.geoStoreClient = new gxp.plugins.GeoStoreClient({
+                url: (this.geostoreUrl) ? this.geostoreUrl : this.target.geoStoreBaseURL,
+                user: (this.geostoreUser) ? this.geostoreUser : this.target.geostoreUser,
+                password: (this.geostorePassword) ? this.geostorePassword : this.target.geostorePassword,
+                proxy: (this.geostoreProxy) ? this.geostoreProxy:this.target.proxy,
+                listeners: {
+                    "geostorefailure": function(tool, msg){
+                        Ext.Msg.show({
+                                title: "Geostore Exception",
+                                msg: msg,
+                                buttons: Ext.Msg.OK,
+                                icon: Ext.Msg.ERROR
+                        });
+                    }
+                }
+            }); 
+        }
+        var geoStore = this.geoStoreClient;
+        var chartsCategory = {type: 'category', name: 'CHARTS'};
+        geoStore.existsEntity(chartsCategory,
+            function(exists) {
+                if(!exists) {
+                    geoStore.createEntity(chartsCategory, function(categoryID) {
```

Comment:
CategoryInitializer plugin should do this work for you (create category). This can be a plus.